server/mlabns/util/maxmind.py

```diff
@@ -29,16 +30,24 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
 
-def get_ip_geolocation(ip_address):
+def get_ip_geolocation(ip_address, address_family):
     """Returns the geolocation data associated with an IP address from MaxMind.
 
     Args:
         ip_address: A string describing an IP address (v4 or v6).
+        address_family: the ip_address format, either ipv4 or ipv6
 
     Returns:
         A populated GeoRecord if matching geolocation data is found for the
         IP address. Otherwise, an empty GeoRecord.
     """
+    maxmind_file = None
+    if address_family == message.ADDRESS_FAMILY_IPv4:
+        maxmind_file = constants.GEOLOCATION_MAXMIND_CITY_FILE_IPv4
+    elif address_family == message.ADDRESS_FAMILY_IPv6:
+        maxmind_file = constants.GEOLOCATION_MAXMIND_CITY_FILE_IPv6
+
```

Comment:
`if / elif / else log an error and proceed with the error case` is the right way to do things when you are working in a domain that has two right answers and many many wrong ones.